### PR TITLE
OPENEUROPA-656: Fix OpenEuropa Profile technical debt issues	

### PR DESCRIPTION
**Pull request by richardcanoe**
Include openeuropa/custom-artifacts to negate the need to run npm builds to import oe_theme.

### CODE REVIEW

**Comment by ademarco** [comment]
The build here will pass as soon as we remove dependencies from oe_theme.info.yml.

**Comment by ademarco** [comment]
Build will be successful after https://github.com/openeuropa/oe_theme/pull/77 will be merged and a new theme version will be released

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Drupal editor configuration normalization
```
Comment (drupol) [inline .editorconfig]: We should use "OpenEuropa" instead of Drupal.

**Comment by ademarco** [comment]
Good catch @netlooker ! It will be solved in theme release 0.3.1 which is currently building.

**Comment by netlooker** [comment]
Ok, I did the second round on this and it works as expected. Just to mention, the Behat test is picking up the text to test on the homepage from the '<title>' tag.

**Pull request merged**